internal/characters/lauma/skill.go

```diff
@@ -0,0 +1,296 @@
+package lauma
+
+import (
+	"github.com/genshinsim/gcsim/internal/frames"
+	"github.com/genshinsim/gcsim/pkg/core/action"
+	"github.com/genshinsim/gcsim/pkg/core/attacks"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/info"
+	"github.com/genshinsim/gcsim/pkg/enemy"
+	"github.com/genshinsim/gcsim/pkg/modifier"
+)
+
+var skillFrames [][]int
+
+const (
+	skillPressHitmark                 = 16
+	frostgroveSanctuaryFirstHitPress  = 31
+	frostgroveSanctuaryFirstHitHold   = 24
+	frostgroveSanctuaryInterval       = 117
+	skillHoldHitmark                  = 45
+	skillConsumeDew                   = 29
+	skillOffset                       = 0
```

Comment:
Is this actually 0? If it's actually 0 then remove all the offsets and use just `nil` for the offset.